_includes/neurips/2019/presenters.nips.2019.html

```diff
@@ -0,0 +1,146 @@
+<!-- Speakers Grid Section -->
+<section id="speakers" class="bg-light-gray">
+    <div class="container">
+        <div class="row">
+            <div class="col-lg-12 text-center">
+                <h2 class="section-heading">Speakers</h2>
+                <h3 class="section-subheading text-muted"></h3>
+            </div>
+        </div>
+        <div class="grid">
+        {% assign featuredItems = site.tags.oral | sort: "featuredOrder" %} 
+        {% for post in featuredItems %}
+            <div class="cell speakers-item">
+                <a href="{{post.website}}" target="_blank" class="speakers-link">
+                    <div class="speakers-hover">
+                        <div class="speakers-hover-content">
+                            <p>{{post.topic}}</p>
+                            <i class="fa fa-plus fa-3x"></i>
+                        </div>
+                    </div>
+                    <img src="/assets/img/NIPS/Speakers/{{ post.img }}" class="responsive-image img-centered desaturate" alt="">
+                </a>
+                <div class="speakers-caption">
+                    <h4>{{ post.name }}</h4>
+                    <p class="text-muted">{{ post.title }}</p>
+                </div>
+            </div>
+        {% endfor %}
+        </div>
+    </div>
+</section>
+<section>
+    <div class="container">
+        <div class="row">
+            <div class="col-lg-12 text-center">
+                <h2 class="section-heading">Posters</h2>
+                <h3 class="section-subheading text-muted"></h3>
+            </div>
+        </div>
+        <ul>
+            	<li>Development of a hand pose recognition system on an embedded computer using CNNs	Dennis H Núñez Fernández (Universidad Nacional de Ingeniería)*</li>
```

Comment:
Previously we listed presenters and their institutions followed by the names of their papers in order by presenters the first name. Please adjust this for ease of readability. 

See the previous workshop as reference: http://www.latinxinai.org/icml-2019-presenters

Reply:
This list could be auto-populated using python script created by ICML 2019 website chair @pablorp80 found at /_includes/icml/2019/make_presenters.py

If you'd like to use, make a copy in the current workshop _includes directory and consult him with any questions.

Reply:
I already cleaned and created this list in requested order